packages/lesswrong/components/posts/DeleteDraft.jsx

```diff
@@ -12,7 +12,7 @@ class DeleteDraft extends Component {
 
       editMutation({
         documentId: post._id,
-        set: {deletedDraft:true},
+        set: {deletedDraft:true, draft: true},
```

Comment:
Our database currently contains exactly one post with `deletedDraft:true, draft:false`, which is `sgkx6SvtvuDBFAZvv`. We should probably modify that one for consistency/data-integrity.